ccpp/data/GFS_typedefs.meta

```diff
@@ -111,6 +111,14 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[wgrs]
+  standard_name = z_wind
```

Comment:
CF standards have "upward_air_velocity", which we should use unless there is a good reason not to. Yes, I understand that we're using x_wind and y_wind already, but there is no z_wind in CF standards for some reason.

Reply:
Is there a CF name for non-hydrostatic vertical velocity?

Reply:
It is unsmoothed_nonhydrostatic_upward_air_velocity

```
[wgrs]
  standard_name = unsmoothed_nonhydrostatic_upward_air_velocity
  long_name = unsmoothed non-hydrostatic upward air velocity
  units = m s-1
  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
  type = real
  kind = kind_phys
```